src/assets/metadata/aws.json

```diff
@@ -13,13 +13,418 @@
       "displayName": "Region",
       "description": "AWS service endpoints",
       "url": "https://docs.aws.amazon.com/general/latest/gr/rande.html"
+    },
+    {
+      "name": "skip_region_validation",
+      "type": "Boolean",
+      "required": false,
+      "displayName": "Skip region validation",
+      "description": "Whether to skip validating the region.",
+      "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs#skip_region_validation"
+    },
+    {
+      "name": "access_key",
+      "type": "String",
+      "required": false,
+      "displayName": "Access key",
+      "description": "Access key ID",
+      "url": "https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html"
+    },
+    {
+      "name": "secret_key",
+      "type": "String",
+      "required": false,
+      "displayName": "Secret key",
+      "description": "Secret access key",
+      "url": "https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html"
+    },
+    {
+      "name": "token",
+      "type": "String",
+      "required": false,
+      "displayName": "Token",
+      "description": "Session token for validating temporary credentials.",
+      "url": "https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html"
+    },
+    {
+      "name": "allowed_account_ids",
+      "type": "Array",
+      "required": false,
+      "displayName": "Allowed Account IDs",
+      "description": "List of allowed AWS account IDs to prevent you from mistakenly<br>using an incorrect one (and potentially end up destroying a live environment).",
+      "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs#allowed_account_ids"
+    },
+    {
+      "name": "forbidden_account_ids",
+      "type": "Array",
+      "required": false,
+      "displayName": "Forbidden Account IDs",
+      "description": "List of forbidden AWS account IDs to prevent you from mistakenly using an incorrect one<br>(and potentially end up destroying a live environment).",
```

Comment:
`<br>` here but not above in `allowed_account_ids`

Reply:
Done